CHANGELOG.md

```diff
@@ -14,6 +14,11 @@
    Includes --gpus option for specifying number of gpus to utilize when serving models for
    evaluation (currently applicable for vLLM only).  Also includes --merge-system-user-message
    flag to enable Mistral based judge models.
+* `ilab data generate`, `ilab model chat`, and `ilab model evaluate` now
+   accept a `--endpoint-ca-cert` parameter to use custom CA certificates with
+   a custom OpenAI endpoint. The existing `--tls-*` have been renamed to
```

Comment:
The remaining should be in the Breaking Changes section.

Reply:
fixed